modules/plugin/ogr/ogr-jni/pom.xml

```diff
@@ -108,15 +108,13 @@
                       <exclude>${test.exclude.pattern}</exclude>
                     </excludes>
                     <argLine>-Xmx${test.maxHeapSize} ${jvm.opts} -Djava.awt.headless=${java.awt.headless} -Djava.io.tmpdir="${java.io.tmpdir}" -Djava.library.path="${env.GT_GDAL}"</argLine>
-                    <environmentVariables>
-                        <PATH>${env.GT_GDAL}${path.separator}${env.PATH}</PATH>
-                        <DYLD_LIBRARY_PATH>${env.GT_GDAL}${path.separator}${env.DYLD_LIBRARY_PATH}</DYLD_LIBRARY_PATH>
-                        <LD_LIBRARY_PATH>${env.GT_GDAL}${path.separator}${env.LD_LIBRARY_PATH}</LD_LIBRARY_PATH>
-                    </environmentVariables>
```

Comment:
I too need to set these environment variables to make things work. I usually do this in my shell settings anyways so I don't have a strong position on keeping them in the pom. I missed the motivation for taking them out?